src/session.rs

```diff
@@ -930,7 +933,18 @@ impl Bwe {
     }
 
     fn poll_estimate(&mut self) -> Option<Bitrate> {
-        self.bwe.poll_estimate()
+        let estimate = self.bwe.poll_estimate()?;
```

Comment:
This does a `take` internally thus you could be at `self.last_emitted_estimate * 1.0499` and as a result of this take out the value and throw it away. The frequency is high enough that this probably doesn't matter

Reply:
This should be sorted now